### ISSUE TITLE
Style Guide

### ISSUE THREAD

**Issue by RyanCCollins** [grommet/grommet-cms-boilerplate#42]
Let's leave this as a running dialog about stylistic choices so that we discuss our preferences and what the best practices are that we should follow on future projects.

**Comment by RyanCCollins** [comment]
Just curious about the underscore for class methods.  I've been following this pattern since picking up the code, but am just wondering if there is a reason for it?

Here is what the [AirBnb style-guide](https://github.com/airbnb/javascript/tree/master/react#methods) says, not that we always need to follow it.

**Comment by karatechops** [comment]
The styling follows grommet core's styling which essentially adds underscores to a component's methods. In my personal JS projects outside of React, I like the pattern of using underscores for private methods but that doesn't make much sense once you're in the React eco-system. 

I lean towards AirBnB's setup for most styling however it'd be a pain to remove the underscores at this point.

**Comment by RyanCCollins** [comment]
Yeah, I agree with that!!  No sense in making drastic changes this late in the game, We can bookmark these decisions for future projects.
